package/libyui-rest-api.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Jul  7 13:10:52 UTC 2020 - riafarov <riafarov@suse.com>
+
+- Handle MHD result as int or enum depending on libmicrohttpd version
```

Comment:
I'd add a version note like "(libmicrohttpd>=0.9.71 uses enum)" so you know what is used where.

Reply:
Good point, added.